src/FilePositionMap.php

```diff
@@ -86,19 +86,23 @@ public function getEndLine($node) : int {
      * Similar to getStartLine but includes the column
      */
     public function getEndLineCharacterPositionForOffset($node) : LineCharacterPosition {
-        return $this->getLineCharacterPositionForOffset($node);
+        return $this->getLineCharacterPositionForOffset($node->getEndPosition());
```

Comment:
The other `ForOffset` methods take an actual offset, should this one have a different name?

Reply:
Also can `$node` have a type annotation?

Reply:
```php
     /**
      * @param Node|Token $node
      * Similar to getStartLine but includes the column
      */
```

Renamed this method, the original name was incorrect